stylesheets/merge-conflicts.less

```diff
@@ -88,17 +79,18 @@
     background: @dirty-line-color;
     &.cursor-line { background: @dirty-current-color; }
   }
+}
+
+atom-overlay, atom-overlay::shadow {
+  // width: 100%;
```

Comment:
I can't do this: it would apply to *all overlays*, everywhere.

I'll need to send a PR to Atom adding an option to apply custom classes to each created overlay.